conf/messages

```diff
@@ -274,8 +274,10 @@ submissions.shutmucr = Shut
 submissions.arrival = Arrive
 submissions.departure = Depart
 submissions.ducrassociation = Associate DUCR
+submissions.ducrpartassociation = Associate DUCR Part
```

Comment:
Are these messages not used anywhere?  Or are they created from the ucr type?

Reply:
They're created from UCR type, that was missing or not added intentionally. Not sure about this, but right now the request type display that this is the Association for DUCR Part. It has been discussed with BAs about adding specific request type message for DUCR Type